pkg/backup/schedule_exec.go

```diff
@@ -578,6 +584,12 @@ func init() {
 						Measurement: "Jobs",
 						Unit:        metric.Unit_TIMESTAMP_SEC,
 					}),
+					RpoTenantMetric: NewExportedGaugeVec(metric.Metadata{
+						Name:        "schedules.BACKUP.tenant-last-completed-time",
+						Help:        "The unix timestamp of the most recently completed backup by a tenant schedule specified as maintaining this metric",
```

Comment:
This isn't a "tenant schedule" but rather a schedule for backing up a tenant. So the schedule still exists on the host cluster, but backs up the tenant.